pytermgui/ansi_interface.py

```diff
@@ -191,25 +199,6 @@ def __call__(self, text: str, color: ColorType, reset_color: bool = True) -> str
 """`Color` instance to setting background colors"""
 
 
-def screen_size() -> tuple[int, int]:
-    """Get screen size using the os module
-
-    This is technically possible using a method of moving the
-    cursor to an impossible location, and using `report_cursor()`
-    to get where the position was clamped. This however messes
-    with the cursor position and printing gets a bit glitchy."""
-
-    try:
-        width, height = get_terminal_size()
-        return (width, height)
-
-    except OSError as error:
-        if error.errno in (9, 25):
-            return 0, 0
-
-        raise
-
-
```

Comment:
We don't need this function since our imported *get_terminal_size* function from **shutil** does similar but better and more efficiently